### ISSUE TITLE
CERTIFICATE_VERIFY_FAILED

### ISSUE THREAD

**Issue by Garin2-colab** [portu-sim/comfyui_bmab#37]
I've got this error - certificate verify failed. This happened when 'Face detailer' was running.
Any solution that you can recommend?
Thank you!

# ComfyUI Error Report
## Error Details
- **Node Type:** BMAB Face Detailer
- **Exception Type:** urllib.error.URLError
- **Exception Message:** <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)>
## Stack Trace
```
  File "C:\Users\giantstep01\AI\StabilityMatrix\Data\Packages\ComfyUI\execution.py", line 323, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)

  File "C:\Users\giantstep01\AI\StabilityMatrix\Data\Packages\ComfyUI\execution.py", line 198, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)

  File "C:\Users\giantstep01\AI\StabilityMatrix\Data\Packages\ComfyUI\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)

  File "C:\Users\giantstep01\AI\StabilityMatrix\Data\Packages\ComfyUI\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))

  File "C:\Users\giantstep01\AI\StabilityMatrix\Data\Packages\ComfyUI\custom_nodes\comfyui_bmab\bmab\nodes\detailers.py", line 171, in process
    results.append(self.process_faces(bind, bgimg, img2img, model, limit, order, withsam))

  File "C:\Users\giantstep01\AI\StabilityMatrix\Data\Packages\ComfyUI\custom_nodes\comfyui_bmab\bmab\nodes\detailers.py", line 91, in process_faces
    boxes, logits = utils.yolo.predict(bgimg, model, 0.35)

**Comment by Garin2-colab** [comment]
I added a code to bypass the verification for now. Thanks.

**Comment by portu-sim** [comment]
which code do you fix?
